scripts/webpack.config.core.js

```diff
@@ -1,16 +1,27 @@
-const path = require('path')
+import { createRequire } from 'module'
+import { fileURLToPath } from 'url'
 
-module.exports = {
-  entry: path.resolve(`${__dirname}/../src/core.js`),
+// TODO: use `import.meta.resolve()` once it is supported without any
+// experimental flags
+const require = createRequire(import.meta.url)
+const PATH_BROWSERIFY_PATH = require.resolve('path-browserify')
+
+const CORE_FILE = fileURLToPath(new URL('../src/core.js', import.meta.url))
+const DIST_DIR = fileURLToPath(new URL('../dist/', import.meta.url))
+
+const webpackConfig = {
+  entry: CORE_FILE,
   devtool: 'source-map',
   output: {
-    path: path.resolve(`${__dirname}/../dist`),
-    filename: 'index.js',
+    path: DIST_DIR,
+    filename: 'index.cjs',
     library: 'frameworkInfo',
     libraryTarget: 'umd',
```

Comment:
This is keeping UMD as a format for browser consumers, as opposed to using ES modules.
I initially tried with ES modules, but this created a wide array of problems when integrating it with `netlify-react-ui`. Those are listed in https://github.com/netlify/netlify-react-ui/issues/9869.
Keeping it as UMD is a smaller change for browser consumers, which will be simpler to integrate.